<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(43, 43, 43);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>200</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border: 1px solid grey;}
QPushButton::pressed{border: 1px solid blue;color: blue;};
</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_Layout"/>
   </widget>
   <widget class="QLabel" name="Partie_Algo">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Algorithm selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>80</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border: 1px solid grey;}
QPushButton::pressed{border: 1px solid blue;color: blue;};
</string>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="Algo_select">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>50</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;</string>
    </property>
    <item>
     <property name="text">
      <string>Binary tree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fractal stick</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fractal plant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fractal sym√©trique</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Buisson</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feuille</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sapin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dragon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Complexe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Persil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arbre 3D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arbre 3D 2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Partie_Env">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2D Tree generator</string>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>130</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="Partie_Algo_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generation delay</string>
    </property>
   </widget>
   <widget class="QSlider" name="brush_Size_Slider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>70</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Brush size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>540</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border: 1px solid grey;}
QPushButton::pressed{border: 1px solid blue;color: blue;};
</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QFrame" name="verticalFrame_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>793</width>
      <height>553</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(43, 43, 43);</string>
    </property>
    <layout class="QVBoxLayout" name="main_Layout_2"/>
   </widget>
   <widget class="QSlider" name="tree_Size_Slider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>110</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tree size</string>
    </property>
   </widget>
   <widget class="QSlider" name="segment_Size_Slider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QSlider" name="alpha_Slider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="minimum">
     <number>-45</number>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
    <property name="value">
     <number>22</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Angle alpha</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_9">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>510</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_10">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>470</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reduction ratio</string>
    </property>
   </widget>
   <widget class="QSlider" name="ratio_Slider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>490</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_11">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_12">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>510</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QSlider" name="rotation_Slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>721</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-180¬∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_14">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>590</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>180¬∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_15">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>590</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0¬∞</string>
    </property>
   </widget>
   <widget class="QSlider" name="vertical_rotation_Slider">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>29</y>
      <width>20</width>
      <height>521</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_16">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>180¬∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_17">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>550</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-180¬∞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>160</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border: 1px solid grey;}
QPushButton::pressed{border: 1px solid blue;color: blue;};
</string>
    </property>
    <property name="text">
     <string>Generate .obj</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_20">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>210</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Angle beta</string>
    </property>
   </widget>
   <widget class="QSlider" name="beta_Slider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>230</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="minimum">
     <number>-45</number>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
    <property name="value">
     <number>22</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_22">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>45¬∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_23">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-45¬∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_24">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>250</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Angle gamma</string>
    </property>
   </widget>
   <widget class="QSlider" name="gamma_Slider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>270</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="minimum">
     <number>-45</number>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
    <property name="value">
     <number>22</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="taillePinceauLabel_25">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0¬∞</string>
    </property>
   </widget>
   <zorder>verticalFrame_2</zorder>
   <zorder>quitButton</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>Partie_Algo</zorder>
   <zorder>searchButton</zorder>
   <zorder>Algo_select</zorder>
   <zorder>Partie_Env</zorder>
   <zorder>speedSlider</zorder>
   <zorder>Partie_Algo_2</zorder>
   <zorder>brush_Size_Slider</zorder>
   <zorder>taillePinceauLabel</zorder>
   <zorder>resetButton</zorder>
   <zorder>tree_Size_Slider</zorder>
   <zorder>taillePinceauLabel_2</zorder>
   <zorder>segment_Size_Slider</zorder>
   <zorder>taillePinceauLabel_3</zorder>
   <zorder>alpha_Slider</zorder>
   <zorder>taillePinceauLabel_4</zorder>
   <zorder>taillePinceauLabel_9</zorder>
   <zorder>taillePinceauLabel_10</zorder>
   <zorder>ratio_Slider</zorder>
   <zorder>taillePinceauLabel_11</zorder>
   <zorder>taillePinceauLabel_12</zorder>
   <zorder>taillePinceauLabel_8</zorder>
   <zorder>rotation_Slider</zorder>
   <zorder>taillePinceauLabel_13</zorder>
   <zorder>taillePinceauLabel_14</zorder>
   <zorder>taillePinceauLabel_15</zorder>
   <zorder>vertical_rotation_Slider</zorder>
   <zorder>taillePinceauLabel_16</zorder>
   <zorder>taillePinceauLabel_17</zorder>
   <zorder>generateButton</zorder>
   <zorder>taillePinceauLabel_20</zorder>
   <zorder>beta_Slider</zorder>
   <zorder>taillePinceauLabel_22</zorder>
   <zorder>taillePinceauLabel_23</zorder>
   <zorder>taillePinceauLabel_24</zorder>
   <zorder>gamma_Slider</zorder>
   <zorder>taillePinceauLabel_25</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>837</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
